web-app/js/portal/filter/combiner/ShapeFileCqlBuilder.js

```diff
@@ -0,0 +1,14 @@
+Ext.namespace('Portal.filter.combiner');
+
+Portal.filter.combiner.ShapeFileCqlBuilder = Ext.extend(Portal.filter.combiner.FilterCqlBuilder, {
+
+    buildCql: function() {
+
+        var cqlParts = this._visualisedFiltersWithValues().map(function(filter) {
+
+            return filter.getCql();
```

Comment:
The url was getting encoded but it was failing because of a trailing slash. Should be fine now. :+1: